pkgs/top-level/all-packages.nix

```diff
@@ -25526,6 +25519,38 @@ with pkgs;
 
   zuki-themes = callPackage ../data/themes/zuki { };
 
+  ### APPLICATIONS / GIS
```

Comment:
I'm not sure about this grouping, do we do it on any other place? The current structure of all-packages is messy, but I think there should be some kind of standardisation process before actively changing the grouping from top-level categories to something else.

Reply:
Another example: https://github.com/NixOS/nixpkgs/pull/178376/commits/d35fc529cf09512aa28f0d78489e122350f20c53
I think it's inline with comment:
```
/* The top-level package collection of nixpkgs.
 * It is sorted by categories corresponding to the folder names in the /pkgs
 * folder. Inside the categories packages are roughly sorted by alphabet, but
 * strict sorting has been long lost due to merges. Please use the full-text
 * search of your editor. ;)
 * Hint: ### starts category names.
 */
```
Just a small step to improve structure.

Reply:
Just skimmed through the file and your pattern seems to be more common than I expected. Thanks for the explanation.